elliottlib/errata.py

```diff
@@ -635,25 +635,11 @@ def add_jira_bugs_with_retry(advisory_id: int, bugids: List[str], noop: bool = F
         if noop:
             logger.info('Dry run: Would have attached bugs')
             continue
-        results = add_multi_jira_issues(advisory_id, chunk_of_bugs)
-        for i, result in enumerate(results):
-            if result.status_code != 201:
-                rt = add_jira_issue(advisory_id, chunk_of_bugs[i])
-                if rt.status_code != 201:
-                    errata_error = rt.json()
-                    if 'errors' not in errata_error or len(errata_error) != 1:
-                        raise exceptions.ElliottFatalError(f"attach jira bug {chunk_of_bugs[i]} failed with "
-                                                           f"status={rt.status_code} "
-                                                           f"errmsg={rt.json()}")
-                    if 'idsfixed' not in errata_error['errors'] or len(errata_error['errors']) != 1:
-                        raise exceptions.ElliottFatalError(f"attach jira bug {chunk_of_bugs[i]} failed with "
-                                                           f"status={rt.status_code} "
-                                                           f"errmsg={rt.json()}")
-                    for err in errata_error['errors']['idsfixed']:
-                        if 'The issue is filed already in' not in ' '.join(err):
-                            raise exceptions.ElliottFatalError(f"attach jira bug {chunk_of_bugs[i]} failed with "
-                                                               f"status={rt.status_code} "
-                                                               f"errmsg={rt.json()}")
+        try:
```

Comment:
Looks like an unnecessary `try` block here.

Reply:
How will this code react to a jira issue that is already attached to a not shipped advisory?

Reply:
removed try block, it will raise an exception, and I think we will not meet that issue https://github.com/openshift/elliott/pull/423/files#r971642699

Reply:
You might be right, but these status codes do not tell me much. You able to link to how these status codes are produced upstream? https://errata.devel.redhat.com/developer-guide/api-http-api.html is not revealing.

Reply:
I guess this is it https://errata.devel.redhat.com/developer-guide/api-http-api.html#api-error-handling
in general, errata_tool is a wrapper of errata_api, The way we used to do it through the original API should now be simplified through errata_tool